src/app/components/MessageBanner/index.test.tsx

```diff
@@ -104,7 +104,7 @@ describe('MessageBanner', () => {
       );
       const image = screen.getByAltText('');
       expect(image.getAttribute('src')).toEqual(
-        summary.imageUrl.replace('{width}', 'raw'),
+        summary.imageUrl.replace('{width}', '240'),
```

Comment:
You should hard code this value, it's not great practice to use the implementation logic as part of the test

Reply:
Actually thinking about this, is there a reason this isn't being done in the BFF?

Reply:
I think these all come back from the BFF as a list of Curations, so the same data is used to display a normal promo as well as the Billboard/MessageBanner components. Could maybe have a lookup table in the BFF that has the resolution to use based on the curation-type, but I'm not too familiar with how its all setup.